source/expressions.tex

```diff
@@ -1636,12 +1624,26 @@
 The closure type for a \grammarterm{lambda-expression} has a public
 inline function call operator (for a non-generic lambda) or
 function call operator template (for a generic lambda)\iref{over.call}
-whose parameters and return type
-are described by the \grammarterm{lambda-expression}'s
-\grammarterm{parameter-declaration-clause} and \grammarterm{trailing-return-type}
-respectively, and whose
-\grammarterm{template-parameter-list} consists of
+whose parameters are described by the \grammarterm{lambda-expression}'s
```

Comment:
I agree with the gist of the change, but having four "whose" in a row is not optimal.

Maybe say 

"The closure type for a \grammarterm{lambda-expression} has a public
inline function call operator (for a non-generic lambda) or
function call operator template (for a generic lambda)\iref{over.call} with the following properties:
(bulleted list)
its parameters...
its return type is...
etc.
"

Reply:
Done, but maybe we should write the `operator()`'s declaration syntactically? I.e.
"""
The closure type for a \grammarterm{lambda-expression} has a public
function call operator as if declared by

   `template` `<` _template-parameter-list_ `>` _requires-clause_
   `auto` `operator()` `(` _parameter-declaration-clause_ `)` ....

where:
— _parameter-declaration-clause_ is the _lambda-declarator_'s _parameter-declaration-clause_,
...
"""

Reply:
We're trying not to be syntactic in this area.

Reply:
> We're trying not to be syntactic in this area.

Why? There are places where the standard does syntactic transformation, like for `for` loops or a coroutine _funciton-body_. Why not do it to define a closure's `operator()`?